### PR TITLE
Task #48979 [Admin] List All Room

### PR DESCRIPTION
**Pull request by anhlh-3049**
## Related Tickets
- https://edu-redmine.sun-asterisk.vn/issues/48979

## Evidence (Screenshot or Video)
Screenshot_6

### CODE REVIEW

**Comment by anhlh-3049** [comment]
ready

File: app/controllers/admin/rooms_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Admin::RoomsController < Admin::AdminController
+  def index
+    @pagy, @rooms = pagy(Room.all, items: 10)
+  end
+
+  private
+  def room_params
+    params.require(:room)
```
Comment (ngocvt-0484) [inline app/controllers/admin/rooms_controller.rb]: ddặt biến constant trong model cho :name, :price, :description, :status, :user_id, :room_view_id đi e. UPDATABLE_ATTRS

File: app/controllers/admin/rooms_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Admin::RoomsController < Admin::AdminController
+  def index
+    @pagy, @rooms = pagy(Room.all, items: 10)
```
Comment (ngocvt-0484) [inline app/controllers/admin/rooms_controller.rb]: 10 VÔ SETTING

File: app/views/admin/rooms/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          <th><%= t ".id" %></th>
+          <th><%= t ".name" %></th>
+          <th><%= t ".price" %></th>
+          <th><%= t ".status" %></th>
+          <th><%= t ".description" %></th>
+        </tr>
+        </thead>
+        <tbody>
+          <% @rooms.each_with_index do |room, index| %>
+        <tr>
```
Comment (ngocvt-0484) [inline app/views/admin/rooms/index.html.erb]: indent

**Comment by anhlh-3049** [comment]
ready

**Pull request merged**